src/frontend/src/components/authenticateBox/index.ts

```diff
@@ -431,7 +431,11 @@ export const authnTemplates = (i18n: I18n, props: AuthnTemplates) => {
 
         <ul class="c-link-group">
           <li>
-            <button @click=${() => useExistingProps.register()} class="t-link">
+            <button
+              @click=${() => useExistingProps.register()}
+              id="registerButton"
```

Comment:
Why the `id`?

Reply:
I added the id there so that the e2e test button click logic for the register button works the same regardless of the template (`firstTime` / `useExisting`).